app/views/play/modal/AmazonHocModal.coffee

```diff
@@ -8,3 +8,15 @@ module.exports = class AmazonHocModal extends ModalView
 
   events:
     'click #close-modal': 'hide'
+    'mouseup #aws-educate-link': 'onClickAwsEducateLink' # mouseup detects middle click as well
+    'mouseup #aws-alexa-link': 'onClickAwsAlexaLink'
+    'mouseup #aws-future-eng-link': 'onClickAwsFutureEngLink'
+  
+  onClickAwsEducateLink: ->
+    window.tracker?.trackEvent 'Click Amazon link', category: 'Amazon', label: 'aws-educate-link'
+  
+  onClickAwsAlexaLink: ->
+    window.tracker?.trackEvent 'Click Amazon link', category: 'Amazon', label: 'aws-alexa-link'
+    
+  onClickAwsFutureEngLink: ->
+    window.tracker?.trackEvent 'Click Amazon link', category: 'Amazon', label: 'aws-future-eng-link'
```

Comment:
Does this event get successfully recorded before the browser navigates to the new location?

Reply:
Not sure best way to check if it would successfully do network requests, but it does successfully run the JS. I guess deploy it to staging and check the DB?